guidelines/sc/20/visual-presentation.html

```diff
@@ -14,6 +14,8 @@ <h4>Visual Presentation</h4>
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
 
-<p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing. When a specific feature is inapplicable to a writing system, this criterion does not require a mechanism to achieve that feature for content authored in that writing system.</p>
+<p class="note">Content is not required to use these values. The requirement is that a mechanism is available for users to change these presentation aspects. The mechanism can be provided by the browser or other user agent. Content is not required to provide the mechanism.</p>
+
+<p class="note">Writing systems for some languages use different presentation aspects to improve readability and legibility. If a presentation aspect in this success criterion is not used in a writing system to improve accessibility, content in that writing system does not need to use that presentation setting and can conform without it. Authors are encouraged to follow guidance for improving readability and legibility of text in their writing system.</p>
```

Comment:
```suggestion
<p class="note">Writing systems for some languages use different presentation aspects to improve readability and legibility. If a presentation aspect in this success criterion is not used in a writing system, content in that writing system does not need to use that presentation setting and can conform without it. Authors are encouraged to follow guidance for improving readability and legibility of text in their writing system.</p>
```

Reply:
Unfortunately that isn't how conformance works, without normative language to do that.